x/farming/types/params.go

```diff
@@ -10,6 +10,15 @@ import (
 	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
 )
 
+const (
+	// PrivatePlanMaxNumDenoms is the maximum number of denoms in a private plan's
+	// staking coin weights, epoch amount and reward denoms.
+	PrivatePlanMaxNumDenoms = 50
+	// PublicPlanMaxNumDenoms is the maximum number of denoms in a public plan's
+	// staking coin weights, epoch amount and reward denoms
+	PublicPlanMaxNumDenoms = 500
+)
+
```

Comment:
Don't we need to specific these variables somewhere in docs?

Reply:
These constants are for extreme cases and usually users won't even notice that these limits present.
Do you have any idea where to put these? I'm considering creating a new section 'Global constants` in `spec/07_params.md`.

Reply:
I thought the same. Having global constants in `07_params.md` sounds good to me. However, if these variables don't necessarily need to be exposed then it is fine with me not to have them in docs.

Reply:
I added the documentation about those constants.